docs/genai/howto/build-model.md

```diff
@@ -13,7 +13,7 @@ nav_order: 2
 * TOC placeholder
 {:toc}
 
-The model builder greatly accelerates creating optimized and quantized ONNX models that run with ONNX Runtime GenAI.
+The model builder greatly accelerates creating optimized and quantized ONNX models that run with the ONNX Runtime generate() API.
```

Comment:
Can we just directly link to the [GitHub README](https://github.com/microsoft/onnxruntime-genai/blob/main/src/python/py/models/README.md)? Otherwise, we will have to keep updating the docs page and the README whenever there's a new change.

Reply:
Updated for now. Let's look at just having the README next update